code/style.css

```diff
@@ -35,24 +88,133 @@ p {
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  border: 2px solid black;
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
   background: midnightblue;
-
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  border: 2px solid black;
 }
 
-button {
-  padding: 8px 16px;
+.button-reveal {
```

Comment:
I'm happy you use classes only in your CSS. In general, your CSS code is very clean, great job :)